packages/sdk/src/generateSnapshotFromManifest.ts

```diff
@@ -59,6 +59,9 @@ export function generateSnapshotFromManifest(
 
   const getPublicPath = (): string => {
     if ('publicPath' in manifest.metaData) {
+      if (manifest.metaData.publicPath === 'auto' && version) {
+        return new URL(version).origin + '/';
```

Comment:
Do we need to support more sophisicated auto resolution? what if remote is localhost/static/chunks/remoteEntry - and not emitted to the root? 

@zhoushaw @2heal1